### DISCUSSION TITLE
Next pydantic version

### DISCUSSION THREAD

**Discussion by vitalik** [pydantic/pydantic#3712] [Question]
Hey @samuelcolvin ,

I'm planing [dependencies for django-ninja project..](https://github.com/vitalik/django-ninja/pull/310#pullrequestreview-858750127)

And wondering how will you continue versioning ? 

 - 1.10, 1.11, 1.12...
 - or 2.0, 2.1, 2.2....

Thank you

**Comment by NowanIlfideme** [comment]
Well, the next version that has non-breaking changes will be `1.10` and so on, as usual with projects.

While 2.0 is being planned at this point, the API will be breaking from 1.X, so you should pin `pydantic>=1.7.4,<2.0` in your dependencies (for example).
